### PR TITLE
Typo in current collector eqns

### LINKED ISSUE #555: typo in 2d current collector eqns
The conductivity is multiplying the through-cell current density instead of the potential

### PR DESCRIPTION
**Pull request by rtimms**
# Description

Fixes typo in current collector equations

Fixes #555

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Key checklist:

- [x] No style issues: `$ flake8`
- [x] All tests pass: `$ python run-tests.py --unit`
- [x] The documentation builds: `$ cd docs` and then `$ make clean; make html`

You can run all three at once, using `$ python run-tests.py --quick`.

## Further checks: 

- [ ] Code is commented, particularly in hard-to-understand areas
- [ ] Tests added that prove fix is effective or that feature works
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: pybamm/parameters/standard_parameters_lithium_ion.py [hunk 1]
```diff
@@ -272,12 +271,17 @@ def U_p_dimensional(sto, T):
...
+sigma_p_prime = sigma_p * delta
+sigma_cp_prime = sigma_cp * delta ** 2
+sigma_cn_dbl_prime = sigma_cn_prime * delta
+sigma_cp_dbl_prime = sigma_cp_prime * delta
 # should rename this to avoid confusion with Butler-Volmer
-
-alpha = 1 / (sigma_cn * (L_x / L_z) ** 2 * l_cn) + 1 / (
-    sigma_cp * (L_x / L_z) ** 2 * l_cp
+alpha = 1 / (sigma_cn * delta ** 2 * l_cn) + 1 / (
+    sigma_cp * delta ** 2 * l_cp
```
Comment (tinosulzer) [inline pybamm/parameters/standard_parameters_lithium_ion.py]: We should still find a better variable for this that doesn't clash with the Butler-Volmer symmetry factor

**Comment by rtimms** [comment]
yeah it makes the potentials in the current collector as expected. didn’t spot it before as the ref potential was missing from the rescaling of phi_s_cp. now the potential looks like the ocv plus a small spatial correction as expected

**Comment by rtimms** [comment]
also i agree about alpha. will try to think of a more meaningful name...

**Comment by tinosulzer** [comment]
I'd been using \varsigma in latex but that's slightly outside the standard set of letters

**Comment by rtimms** [comment]
fixed by #564

**Pull request closed without merging**